llpc/lower/llpcSpirvLowerGlobal.cpp

```diff
@@ -425,10 +425,9 @@ void SpirvLowerGlobal::handleLoadInstGEP(GetElementPtrInst *const getElemPtr, Lo
          "Chained GEPs should have been coalesced by SpirvLowerAccessChain.");
   assert(inOut && "The root of the GEP should always be the global variable.");
 
-  for (auto i = getElemPtr->idx_begin() + 1, e = getElemPtr->idx_end(); i != e; ++i)
+  for (auto i = getElemPtr->idx_begin() + 1, end = getElemPtr->idx_end(); i != end; ++i)
```

Comment:
nit: the standard llvm convention is to call the end value `e`. For very short loops where we only use the end value in the loop condition, I don't think there's much point being verbose

Reply:
Actually, I think you can just use `for (auto index : getElemPtr->indices())` to avoid the end value all together.

Reply:
Actually, we want to avoid the first index since for GEP it is always 0. So I don't use `for (auto index : getElemPtr->indices())`. I will revert it back to "e" :)

Reply:
You can do `drop_begin(getElemPtr->indices())`

https://llvm.org/doxygen/namespacellvm.html#a02981de53fb6ffd384d39addc4d25f37

Reply:
Oh. That's very good.